libs/hdf-converters/src/prisma-mapper.ts

```diff
@@ -0,0 +1,189 @@
+import {ExecJSON} from 'inspecjs';
+import _ from 'lodash';
+import {version as HeimdallToolsVersion} from '../package.json';
+import {
+  BaseConverter,
+  ILookupPath,
+  MappedTransform,
+  parseCsv
+} from './base-converter';
+
+export type PrismaControls = {
+  records: Record<string, string>[];
+};
+
+export type PrismaControl = {
+  Packages: string;
+  Description: string;
+  Distro: string;
+  Type: string;
+  Hostname: string;
+  'Fix Status'?: string;
+  Cause?: string;
+};
+
+const SEVERITY_LOOKUP: Record<string, number> = {
+  low: 0.3,
+  moderate: 0.5,
+  high: 0.7,
+  important: 0.9,
+  critial: 1
+};
+
+const DEFAULT_NIST_TAG = ['SA-11', 'RA-5'];
+const BASE_NIST_TAG = ['SI-2', 'RA-5'];
```

Comment:
Please differentiate between these two (either in a comment or by variable name)